source/Server/Configuration/LdapConfigurationSettings.cs

```diff
@@ -31,9 +31,10 @@ public override IEnumerable<IConfigurationValue> GetConfigurationValues()
             yield return new ConfigurationValue<bool>("Octopus.WebPortal.LdapIgnoreSslErrors", ConfigurationDocumentStore.GetIgnoreSslErrors(), isEnabled, "Ignore SSL errors");
             yield return new ConfigurationValue<string>("Octopus.WebPortal.LdapUsername", ConfigurationDocumentStore.GetConnectUsername(), isEnabled, "Username");
             yield return new ConfigurationValue<SensitiveString>("Octopus.WebPortal.LdapPassword", ConfigurationDocumentStore.GetConnectPassword(), isEnabled, "Password");
-            yield return new ConfigurationValue<string>("Octopus.WebPortal.LdapBaseDn", ConfigurationDocumentStore.GetBaseDn(), isEnabled, "Base DN");
-            yield return new ConfigurationValue<string>("Octopus.WebPortal.LdapDefaultDomain", ConfigurationDocumentStore.GetBaseDn(), isEnabled, "Default Domain");
+            yield return new ConfigurationValue<string>("Octopus.WebPortal.LdapUserBaseDn", ConfigurationDocumentStore.GetUserBaseDn(), isEnabled, "User Base DN");
+            yield return new ConfigurationValue<string>("Octopus.WebPortal.LdapDefaultDomain", ConfigurationDocumentStore.GetDefaultDomain(), isEnabled, "Default Domain");
```

Comment:
Incorrect value was being set for this configuration setting previously

Reply:
Good pickup! 👍